htdocs/adherents/class/adherent.class.php

```diff
@@ -176,8 +176,20 @@ class Adherent extends CommonObject
 
     public $photo;
 
-	public $datec;
-	public $datem;
+	/**
+     * Date creation record (datec)
+     *
+     * @var DateTime
```

Comment:
DateTime is a PHP object
Here $datem are variable with type timestamp, so it should be "integer".
@var integer
Note that sometimes the field can be '' so type should be in doc
@var integer|string